src/templates/ja/2019/featured_chapters.html

```diff
@@ -10,72 +10,72 @@
   {%- set featured_chapter_stats = {"stat1":"89%","label1":"Sites with more 3P code that 1P","stat2":"83%","label2":"Sites that use jQuery","stat3":"4.6%","label3":"Home pages using React"} %}
 {%- elif featured_chapter == "css" %}
   {%- set featured_chapter_name = "CSS" %}
-  {%- set featured_chapter_quote = "Cascading Style Sheets (CSS) are used to paint, format, and layout web pages. Their capabilities span concepts as simple as text color to 3D perspective. It also has hooks to empower developers to handle varying screen sizes, viewing contexts, and printing. CSS helps developers wrangle content and ensure it's adapting properly to the user." %}
-  {%- set featured_chapter_stats = {"stat1":"5%","label1":"Pages using custom properties","stat2":"2%","label2":"Sites that use CSS Grid","stat3":"780","label3":"Number of digits in largest Z-Index value"} %}
+  {%- set featured_chapter_quote = "Cascading Style Sheets (CSS) は、Web ページのペイント、フォーマット、レイアウトに使用されます。その機能は、テキストの色から3Dパースペクティブと、シンプルな概念にまで及びます。また、さまざまな画面サイズ、表示コンテキスト、および印刷を処理するために開発者に力を与えるフックを持っています。CSS は、開発者がコンテンツを調整し、ユーザーに適切に適応することを確認するのに役立ちます。" %}
+  {%- set featured_chapter_stats = {"統計1":"5%","ラベル1":"Pages using custom properties","統計2":"2%","ラベル2":CSS Gridを使用しているサイト"","統計3":"780","ラベル3":"Z-Indexの最大値の桁数"} %}
```

Comment:
The statX and labelX are all variables so need to leave them as they are originally. Can you change them all back for all lines?

Also this CSS line has a Quote issue with label2 - fixed below. The rest don't have this issue and look good.

```suggestion
  {%- set featured_chapter_stats = {"stat1":"5%","label1":"Pages using custom properties","stat2":"2%","label2":"CSS Gridを使用しているサイト","stat3":"780","label3":"Z-Indexの最大値の桁数"} %}
```

Reply:
Thanks, I fix.